src/panels/config/script/ha-script-trace.ts

```diff
@@ -0,0 +1,502 @@
+import {
```

Comment:
This file is mostly identical to `src/panels/config/automation/ha-automation-trace.ts`
Instead of all these duplicates, we should make this more generic as `ha-trace.ts`

Reply:
I intentionally didn't do that for this part, all the rest I generalized, just like the script editor and automation are pretty similar but separate files. It allows us to add diverge more easily later.